packages/outstatic/src/utils/escapeRegExp.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Escapes a string for a use in regular expressions.
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping}
+ */
+export const escapeRegExp = (string: string): string => {
```

Comment:
This could come from `@tiptap/core`, but it's undocumented, so I am not sure if it should be treated as a public API. We could also download some helpers package like lodash, but that tree-shakes better.

Reply:
Let's not rely on importing this from `@tiptap/core`, they might change it in the future. Thanks for this update.